droid-container/pom.xml

```diff
@@ -102,8 +102,8 @@
             <artifactId>xercesImpl</artifactId>
         </dependency>
         <dependency> <!--JAXB-API-->
```

Comment:
The comment is now wrong. Actually, the comment was a bit pointless as it just says the "artifactId", it can simply be removed.

Reply:
Good spot, I've removed this one and the other two.